coil-compose-base/src/main/java/coil/compose/AsyncImagePainter.kt

```diff
@@ -247,17 +251,19 @@ class AsyncImagePainter internal constructor(
     internal var contentScale = ContentScale.Fit
     internal var filterQuality = DefaultFilterQuality
     internal var isPreview = false
+    internal val requestInternal = MutableStateFlow(request)
```

Comment:
Is the a performance benefit to reading request from a `MutableStateFlow` vs. a `mutableStateOf`?